### PR TITLE
pfetch: add page

### PR DESCRIPTION
**Pull request by glapa-grossklag**
Closes issue #6352.

- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by navarroaxel — APPROVED** [review]

File: pages/common/pfetch.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+> Display system information.
+> More information: <https://github.com/dylanaraps/pfetch>.
+
+- Display only the ASCII art and color palette fields:
+
+`PF_INFO="{{ascii palette}}" pfetch`
+
+- Display all possible fields:
+
+`PF_INFO="{{ascii title os host kernel uptime pkgs memory shell editor wm de palette}}" pfetch`
```
Comment (sbrl) [inline pages/common/pfetch.md:12]: There isn't a special keyword `all` or anything?

**Review by sbrl — APPROVED** [review]
Hey, thanks for the new page @glapa-grossklag! Just got 1 minor comment below, but otherwise looks good :D

File: pages/common/pfetch.md (hunk 1 above)
Comment (pixelcmtd) [reply pages/common/pfetch.md:12]: `PF_INFO=all` results in an empty output.

**Review by pixelcmtd — COMMENTED** [review]
I think, it would be better to have just the plain `pfetch` as the first example.

**Comment by navarroaxel** [comment]
> I think, it would be better to have just the plain `pfetch` as the first example.

@pixelcmtd Oh, is a plain `pfetch` useful ? Please send the PR

**Comment by pixelcmtd** [comment]
I've never used it with any options or flags, why would I?

**Pull request merged**